src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1182,6 +1161,46 @@ components:
         message_hex:
           $ref: "#/components/schemas/HexString"
 
+    #
+    # /stream/* Request & Response Types
+    #
+
+    StreamTransactionsRequest:
+      allOf:
+        - $ref: "#/components/schemas/AtLedgerStateMixin"
+        - $ref: "#/components/schemas/CursorLimitMixin"
+        - type: object
+          properties:
+            transaction_type:
+              type: string
+              enum:
+                - user
+                - epoch_change
+                - all
+            from_ledger_state:
+              $ref: "#/components/schemas/LedgerStateSelector"
+            order:
+              type: string
```

Comment:
should we mention what's the default ordering?

Reply:
Good catch!